src/modules/emote_menu/stores/twitch-emotes.js

```diff
@@ -105,7 +118,23 @@ export async function loadTwitchEmotes() {
   const isDark = settings.get(SettingIds.DARKENED_MODE);
   const tempSets = {};
 
-  for (const {owner, id, emotes} of data.channel.self.availableEmoteSets) {
+  const {user} = data;
+  const concatSubscriptionProducts = [];
+
+  for (const set of data.user.subscriptionProducts) {
+    for (const emote of set.emotes) {
+      emote.locked = true;
+      concatSubscriptionProducts.push(emote);
+    }
+  }
+
+  const subscriptionProducts = {
+    id: user.id,
+    owner: user,
+    emotes: concatSubscriptionProducts,
+  };
+
+  for (const {owner, id, emotes} of [...data.channel.self.availableEmoteSets, subscriptionProducts]) {
```

Comment:
what happens if a user is subscribed to the emote set already? is there duplicates? do we override?

Reply:
We override it, as seen in the second image

Reply:
is this a reliable mechanism? I think this relies on `uniqby`